### DISCUSSION TITLE
Is it possible to make the url in Graphqlrc.json a variable?

### DISCUSSION THREAD

**Discussion by TvanderNol** [ChilliCream/hotchocolate#6193] [Q&A]
I have multiple GraphQL API's that I would like to use in the same application (sometimes, it has to use one, another time the other API). It is not possible to make this one API. Is it possible to make the URL in the Graphqlrc.json file configurable as a variable so that it is 'easier' to switch between APIs? The only solution I found was the tooling (`dotnet graphql update -u|--uri`).

**Comment by monsterCoder1012** [comment]
Has there been any updates on this functionality?
